src/game.cpp

```diff
@@ -3973,10 +3923,7 @@ Uint32 timerCallback(Uint32 interval, void* param)
 
 void startMessages()
 {
-	newString(&messages, 0xFFFFFFFF, language[734], stats[clientnum]->name);
```

Comment:
We should have a replacement for this somewhere in the UI

Reply:
Perhaps, these lines use input strings and are arguably not used in the UI redesign? so I deprecated them completely, but we can always add them back sometime in some way shape or form